Styles/main.css

```diff
@@ -211,7 +211,7 @@ main {
 .col-span-2 {
     grid-column: span 2;
 }
-.responsive {
+img {

```

Comment:
Just use the `img` selector so that ALL images on the site are flexible by default.  (If you want some images to NOT be flexible this can be done using more specific selectors.)